aioshelly/wsrpc.py

```diff
@@ -231,7 +231,7 @@ async def call(
 
         if "result" in resp:
             _LOGGER.debug("%s(%s) -> %s", call.method, call.params, resp["result"])
-            return resp["result"]
+            return dict(resp["result"])
```

Comment:
I think we should use `cast()` instead of `dict()`.

Reply:
Fixed